airgun/views/template.py

```diff
@@ -0,0 +1,42 @@
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+
+from airgun.views.common import BaseLoggedInView
+from airgun.widgets import ActionsDropdown
+from airgun.widgets import RadioGroup
+
+
+class SyncTemplatesView(BaseLoggedInView):
+    title = Text("//h2[contains(., 'Import or Export Templates')]")
+    import_export = RadioGroup("//div[@class='col-md-6']")
+    submit = Text("//button[@type='submit']")
```

Comment:
```suggestion
    submit = Text("//button[@type='submit']")
    cancel = Text('//a[text()="Cancel"]')
```

Reply:
I think this needs to be added as there is some UI flow out there, which navigates to 
![Import from https   github com theforeman community-templates git](https://user-images.githubusercontent.com/3190629/77942210-eb099e80-72d8-11ea-9166-abec02895c9d.png)

I have one conflict here to mention that in Satellite we are having the all entities view by default on navigation from the menu bar, but in this case it actually creates entity page on the navigation. (this really different than the overall design we have in Satellite). This is unrelated to this PR by the way, I'm just expressing my thoughts.

Reply:
This is a reports page basically which is not necessarily to be implemented as the verification can be done from templates pages directly. But I understood that this has to be implemented to test UI reports of export/import sometimes.